packages/video_player/video_player_avfoundation/darwin/video_player_avfoundation/Sources/video_player_avfoundation/FVPVideoPlayerTextureApproach.m

```diff
@@ -0,0 +1,164 @@
+#import "FVPVideoPlayerTextureApproach.h"
+
+@implementation FVPVideoPlayerTextureApproach
```

Comment:
This code was a part of `FVPVideoPlayer` class. It all related to texture approach. I extracted it from `FVPVideoPlayer` and put here. This class extends `FVPVideoPlayer` and adds texture approach specific code.